src/django_healthchecks/checker.py

```diff
@@ -57,7 +60,9 @@ def _get_check_functions(name=None, request=None):
         if name and name != service:
             continue
 
-        if callable(func_string):
+        if func_string.startswith(('https://', 'http://')):
+            check_func = _http_healthcheck_func(func_string)
+        elif callable(func_string):
```

Comment:
If func_string is a callable then the startswith attribute isn't available existing